tests/unit/data/test_storage.py

```diff
@@ -38,6 +39,9 @@ def test_DataStoreException_is_subclass_of_CleanAirFrameworkException(self):
     def test_AURNSiteDataStoreException_is_subclass_of_DataStoreException(self):
         self.assertTrue(issubclass(AURNSiteDataStoreException, DataStoreException))
 
+# TODO: Get mock_s3 working so I can reinstate these tests
+# I believe this is a conda issue in that we seem to all be running on
+# different environments.  Let's get this sorted.
```

Comment:
👍🏻

Reply:
I may well include this point in the code review.  It seems somewhat relevant.